tests/unit/dojson/test_dojson_hep_bd2xx.py

```diff
@@ -170,3 +170,68 @@ def test_titles_from_245_and_247__a_9():
     result = hep2marc.do(result)
 
     assert expected == result['245']
+
+
+def test_title_translations_from_242__a():
```

Comment:
This test is wrong: if there is a `242`, `245` goes to `translated_titles` with the right language, whereas `242` goes to titles.